tests/foreman/ui/test_organization.py

```diff
@@ -27,6 +27,31 @@
 from robottelo.manifests import original_manifest
 from robottelo.manifests import upload_manifest_locked
 
+CUSTOM_REPO_ERRATA_ID = settings.repos.yum_0.errata[0]
+
+
+@pytest.fixture(scope='module')
+def module_repos_col(module_org, module_lce, module_target_sat, request):
+    upload_manifest_locked(org_id=module_org.id)
+    repos_collection = module_target_sat.cli_factory.RepositoryCollection(
```

Comment:
@shweta83 Could you check if this is expected?

Reply:
@Gauravtalreja1 No. This fixture has been added in 6.10+ branch so it is not applicable for 6.9.z. I am going to close this PR.